### PR TITLE
Fix stamp sound

### PR DESCRIPTION
**Pull request by crazybrain23**
## About the PR

This makes them sound quieter by making it a mono track. Not sure why the file size got bigger though. Something to do with it not being positional sound with a stereo track.

**Media**

https://github.com/space-wizards/space-station-14/assets/44417085/fb988bea-5faa-4f7c-b696-be29dd955cd3

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: crazybrain
- fix: Stamps are quieter than before.

### CODE REVIEW

**Comment by crazybrain23** [comment]
Before this it just played at full volume at any range through any obstacles. I did try adding AudioParams for the volume but I got an error.

**Comment by DogZeroX** [comment]
How did you miss this? There's usually a fatal error when you use stereo.

**Comment by crazybrain23** [comment]
> How did you miss this? There's usually a fatal error when you use stereo.

I didn't add the sound originally, but the fatal error is it found I found the issue. Did #17834 not use the dev environment?

**Comment by crazybrain23** [comment]
I figured out how to actually make it quieter.

File: Content.Server/Paper/PaperSystem.cs [hunk 1]
```diff
@@ -122,7 +123,7 @@ private void OnInteractUsing(EntityUid uid, PaperComponent paperComp, InteractUs
                 var stampPaperSelfMessage = Loc.GetString("paper-component-action-stamp-paper-self", ("target", Identity.Entity(args.Target, EntityManager)), ("stamp", args.Used));
                 _popupSystem.PopupEntity(stampPaperSelfMessage, args.User, args.User);
 
-                _audio.PlayPvs(stampComp.Sound, uid);
+                _audio.PlayPvs(stampComp.Sound, uid, AudioParams.Default.WithVolume(-2f).WithMaxDistance(5));
```
Comment (metalgearsloth) [inline Content.Server/Paper/PaperSystem.cs:126]: These params should be on the stamcomp sound itself.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Pull request merged**